### PR TITLE
Changes to test1 for github tutorial

### PR DESCRIPTION
**Pull request by ccallaghan-configit**
Leave comments

* Unordered list
* Some `code` practice

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # test1
 test1
+I have made changes
```
Comment (mfaux) [inline README.md:3]: Were these changes intentional? @ccallaghan-configit

**Review by mfaux — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ccallaghan-configit) [reply README.md:3]: Yes, yes they were.
